src/triage/util/pickling.py

```diff
@@ -0,0 +1,23 @@
+import pickle
+
+
+def can_pickle(obj):
+    """Check whether or not an object can be pickled by attempting to pickle it to
+    a BytesIO
+
+    Since it pickles to an in-memory object, be wary about using on huge objects!
```

Comment:
This last bit is true, but it's just a `str` – or rather, in Py3k lingo – a `bytes`, not a `BytesIO`, right?

Reply:
Anyway that seems like a kind of technical implementation detail, so perhaps this could just be a short-and-sweet docstring:

```
Return whether or not an object can be pickled.

Attempts to pickle the given argument in memory via the `pickle` module.
Since it pickles to memory, be wary about using it on huge objects!
```

Reply:
Sure.  However given the other comment, if I'm using the FOrkingPickler directly, I probably don't want to create this new module....I'm not generically testing pickling, I'm testing the multiprocessing Pickler. I should just move this try block into MultiCore directly. Sound right?

Reply:
Sure, sounds good to me